earth_enterprise/src/fusion/autoingest/sysman/AssetVersionD.cpp

```diff
@@ -300,6 +344,7 @@ void
 AssetVersionImplD::HandleChildStateChange(const std::string &) const
 {
   // NoOp in base since leaves don't need to do anything
+  notify(NFY_VERBOSE, "HandleChildStateChange: %s", GetRef().c_str());
 }
 
 void
```

Comment:
I apologise, I should have been more specific. I also meant to say you should differentiate